src/XCCDF/public/xccdf_benchmark.h

```diff
@@ -773,6 +773,13 @@ OSCAP_DEPRECATED(int xccdf_result_export(struct xccdf_result *result, const char
 struct oscap_source *xccdf_result_export_source(struct xccdf_result *result, const char *filepath);
 
 /**
+ * Export TestResult to oscap_source structure
+ * @memberof xccdf_result
+ * @returns newly created oscap_source or NULL on error
+ */
+struct oscap_source *stig_rule_id_xccdf_result_export_source(struct xccdf_result *result, const char *filepath);
```

Comment:
This is a method so please name it accordingly, it should start with the "class" prefix -  xccdf_result_